examples/mnist_template.py

```diff
@@ -37,13 +37,24 @@ class MNISTStepLRConf(StepLRConf):
     step_size: int = 1 
 
 @dataclass
-class MNISTNetConf:
+class MNISTModelConf:
+    # @package _group_ #is this how packages are done?
```

Comment:
If you have questions, there a better channel than comments in pull requests to ask them.

Reply:
old comment that snuck in there, didn't intend for it to be a part of the PR 🙂